front/src/applications/operationalStudies/components/Study/StateStep.tsx

```diff
@@ -1,14 +1,17 @@
+import React from 'react';
 import cx from 'classnames';
 import { patch } from 'common/requests';
 import { useTranslation } from 'react-i18next';
 import { PROJECTS_URI, STUDIES_URI } from '../operationalStudiesConsts';
 
+type StateType = 'started' | 'inProgress' | 'finish';
```

Comment:
could it be an enum ?

Reply:
Tried but typing that with enum does'nt match (or I don't know how to make it match) the typed returned by the openApi.yaml in osrdEditoastApi.ts line 1040.